conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
@cyphar Actually I'm telling runC to run one container with a given bundle path and runC explicitly ignores the second argument. It does not interpret its second argument as another container name and [only uses the first argument as a container name](https://github.com/opencontainers/runc/blob/master/utils_linux.go#L283).
AFAICT the `runc create arg1 arg2` semantics are not defined anywhere in runc, and `arg2` is neither interpreted as a container name nor as a bundle path. Am I missing something from the runC code ?